docs/guides/developer/docs/architecture/api/lifecycle-management-api.md

```diff
@@ -0,0 +1,135 @@
+[TOC]
+
+# Information
+
+The LifeCycle Management API is available since API version 1.12.0.
+
+### GET /api/lifecyclemanagement/policy/{id}
```

Comment:
I do not want to start a singular vs plural debate in REST API design. There are valid arguments on both sides ;). It's important to have consistent API, and therefore, I would change all `/api/lifecyclemanagement/policy/` paths to `/api/lifecyclemanagement/policies`.